src/mscorlib/src/System/Span.cs

```diff
@@ -243,12 +244,42 @@ public void Clear()
         /// <summary>
         /// Fills the contents of this span with the given value.
         /// </summary>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
This should not be marked aggressive inlining - it is too big of a method to inline.